tests/smoketest/minimal-collectd.conf.template

```diff
@@ -11,14 +11,14 @@ LoadPlugin cpu
 LoadPlugin amqp1
 <Plugin "amqp1">
   <Transport "name">
-    Host "qdr-white.sa-telemetry.svc.cluster.local"
+    Host "saf-default-interconnect.sa-telemetry.svc.cluster.local"
```

Comment:
Should be able to use the short DNS name here too to avoid hard coded namespace?

Reply:
yes of course. That was an oversight on my side. Sorry.